engine/src/main/java/org/hibernate/search/query/dsl/impl/ConnectedMultiFieldsRangeQueryBuilder.java

```diff
@@ -67,11 +67,11 @@ public Query createQuery() {
 			return queryCustomizer.setWrappedQuery( createQuery( fieldContexts.get( 0 ), conversionContext ) ).createQuery();
 		}
 		else {
-			BooleanQuery aggregatedFieldsQuery = new BooleanQuery();
+			BooleanQuery.Builder builder = new BooleanQuery.Builder();
```

Comment:
Actually this is my doubt. On one side, I admit that `builder` is confusing, especially in complex code, because we lose visibility about its utility (e.g. aggregation) or type (e.g. boolean). On the other side, the naming with 4 words seems to be difficult to read. @gsmet, could you share your vision about it ?